dev/src/index.ts

```diff
@@ -401,7 +401,8 @@ export class Firestore {
 
         logger('Firestore', null, 'Initialized Firestore GAPIC Client');
         return client;
-      }
+      },
+      /* clientDestructor= */ (client: GapicClient) => client.close()
```

Comment:
hmmm - do we really have `.close()`? :)

Reply:
Not yet - but this PR shows how we could add it. We still have to add it to the Gapic generator though (if you are okay with the general direction).

Reply:
@alexander-fenster Do you know what the next sets of steps would be if we wanted to add this support?